x-pack/qa/sql/src/main/resources/unsupported.sql-spec

```diff
@@ -0,0 +1,28 @@
+//https://github.com/elastic/elasticsearch/issues/31863
```

Comment:
I think the file should be `case_functions.sql-spec` or something and we should include this file in the test case only if we're in a locale the h2 is fine with.

Reply:
Or, better, we should include them in the test but skip them. I think that is better from a reporting standpoint.

Reply:
@nik9000 can we use rest blacklist to blacklist them?

Reply:
I think something like
```
if (name.startsWith(unsupported)) {
  assumeTrue(goodLocale);
}
```

Would be better. I don't like changing the tests we run based on the locale but I don't mind skipping them based on locale.

Reply:
:+1: I like it!